src/Proxies/WP.php

```diff
@@ -340,4 +340,13 @@ public function wp_add_inline_script( string $handle, string $data, string $posi
 	public function wp_print_inline_script_tag( string $data, array $attributes = [] ) {
 		return wp_print_inline_script_tag( $data, $attributes );
 	}
+
+	/**
+	 * Determines whether WordPress is currently serving a REST API request.
+	 *
+	 * @return bool True if it's a WordPress REST API request, false otherwise.
+	 */
+	public function wp_is_serving_rest_request(): bool {
+		return wp_is_serving_rest_request();
```

Comment:
@mukeshpanchal27 based on the ticket description should we be checking the function is available and if not fallback to the old method? e.g

```suggestion
		if ( function_exists( 'wp_is_serving_rest_request' ) ) {
				return wp_is_serving_rest_request();
		}
		
		return ( defined( 'REST_REQUEST' ) && REST_REQUEST );
```

Reply:
@jjgrainger, in #2998 we bumped the WordPress version requirement to 6.6, and since that function was introduced in WP 6.5, it will always be available. So there's no need to wrap it with `function_exists` anymore